src/linux/backend-v4l2.cpp

```diff
@@ -1911,9 +1911,9 @@ namespace librealsense
                 return range;
             }
 
-            struct v4l2_queryctrl query = {};
+            struct v4l2_query_ext_ctrl query = {};
             query.id = get_cid(option);
-            if (xioctl(_fd, VIDIOC_QUERYCTRL, &query) < 0)
+            if (xioctl(_fd, VIDIOC_QUERY_EXT_CTRL, &query) < 0)
```

Comment:
Will this affect other controls as well?

Does this means this comment will be no longer relevant?

```
// Some controls (exposure, auto exposure, auto hue) do not seem to work on V4L2
                // Instead of throwing an error, return an empty range. This will cause this control to be omitted on our UI sample.
                // TODO: Figure out what can be done about these options and make this work
```

Reply:
@hsuys ?

@dmipx  can you review this change as well?
It should work with Ubuntu and JetPack w/o any regressions